packages/web-wallet/src/state/submitDapp/reducer.js

```diff
@@ -0,0 +1,40 @@
+import { merge } from 'state/utils';
+import { createActions, createReducer } from 'redux-yo';
+
+export const submitSteps = {
+  completed: 'completed',
+  createAccount: 'createAccount',
+  detailsForm: 'detailsForm',
+};
+
+export const submitDappActions = createActions(
+  [
+    'createUserAccount',
+    'mergeUserAccount',
+    'reset',
+    'setSubmitStep',
+    'setUserAccount',
+    'submitDapp',
+  ],
+  'submitDapp'
+);
+
+const initialState = {
+  isAppSubmitted: false,
+  submitStep: submitSteps.createAccount,
```

Comment:
After DApp submission `submitStep` changes to `completed` and stays like that until the next page refresh, so a user cannot submit new DApp. It seems we should reset `submitStep` at some point.